install_executorch.py

```diff
@@ -54,6 +54,15 @@ def clean():
     print("Cleaning buck cached state and metadata ...")
     shutil.rmtree(os.path.expanduser("~/.buck/buckd"), ignore_errors=True)
 
+    tokenizer_dirs = [
+        "extension/llm/tokenizers/build",
+        "extension/llm/tokenizers/pytorch_tokenizers.egg-info",
+    ]
+
+    for d in tokenizer_dirs:
+        print(f"Cleaning {d}...")
+        shutil.rmtree(d, ignore_errors=True)
+
```

Comment:
Thank you for adding this, can you also add to Makefile? https://github.com/pytorch/executorch/blob/main/Makefile#L197